docker-compose.yml

```diff
@@ -62,17 +61,20 @@ services:
     depends_on:
       - bootstrap
 
-  waku-publisher:
-    image: alrevuelta/waku-publisher:9fb206c
-    entrypoint: sh
-      - 'opt/run_wakupublisher.sh'
+  rest-traffic:
+    build:
+      dockerfile: Dockerfile.rest-traffic
+    command: >
+      python /opt/traffic.py
+      -mn=http://waku-simulator_nwaku_[1..$NUM_NWAKU_NODES]:8645
```

Comment:
minor nitpick but can we show the show the whole flag name? perhaps easier to see its meaning.